en/docs/administer/configuring-a-read-only-ldap-user-store.md

```diff
@@ -147,31 +141,41 @@ user store:
         <Property name="ConnectionName">uid=AdminLDAP,ou=system</Property>
         <Property name="ConnectionPassword">2010#Avrudu</Property>
         ```
+        ```toml
+        [user_store.properties]
+        "ConnectionName" = "uid=AdminLDAP,ou=system"
```

Comment:
Should these properties be within double quotes? If not, please change in all relevant places.